app/src/main/java/com/ncnf/database/DatabaseResponse.java

```diff
@@ -17,7 +17,11 @@ public boolean isSuccessful() {
         return isSuccessful;
     }
 
-    public Object getResult() {
+    public T getResult() {
+        return result;
+    }
+
+    public T getResult(Class type) {
         return result;
```

Comment:
I guess it won't be an issue once this class' gone, but those two methods appear to do the same thing

Reply:
Yep, this is deleted.